### ISSUE TITLE
Webhook endpoint to flush the cache

### ISSUE THREAD

**Issue by johnbillion** [jazzsequence/dashboard-changelog#1]
Thinking out loud here. If the cache is set to 24 hours you could see stale release notes for quite a while after a new release. It  would be neat if the plugin provided a webhook endpoint which flushes the cache, and then in your GitHub repo you can set up a webhook which triggers on a new release. That way, when a new release is published it'll flush the cache and you'll always see the freshest release straight away.

**Comment by jazzsequence** [comment]
Good idea. For what I'm using it for personally, I don't mind much if the updates are a little stale because chances are I'm not logging in every day anyway, but I can definitely see this being useful for a larger site with more frequent updates.
